.kitchen.dokken.yml

```diff
@@ -31,62 +35,55 @@ platforms:
       pid_one_command: /sbin/init
       intermediate_instructions:
         - RUN yum -y install which initscripts sudo
+    attributes:
+      poise-service:
+        hashicorp-vault:
+          provider: upstart
+  - name: centos-5
+    named_run_list: centos
+    driver:
+      image: centos:5
+      platform: rhel
+      pid_one_command: /sbin/init
+      intermediate_instructions:
+        - RUN yum -y install which initscripts sudo
   - name: ubuntu-16.04
     named_run_list: debian
     driver:
       image: ubuntu:16.04
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN /usr/bin/apt-get update -y
-        - RUN /usr/bin/apt-get install sudo -y
+        - RUN /usr/bin/apt-get install sudo net-tools -y
     attributes:
       poise-service:
-        consul:
+        hashicorp-vault:
```

Comment:
As I've understood, if we wanna override the provider name, then we have to set it via attribute containing the service name. According to `attributes/default.rb`, the service name is `vault`, not `hashicorp-vault`. Probably, that's the reason why Ubuntu 16.04 suite is failing in dokken.